core/org.openjdk.jmc.common/src/main/java/org/openjdk/jmc/common/unit/TimestampKind.java

```diff
@@ -140,8 +140,8 @@ public String format(IQuantity q) {
 	}
 
 	static {
-		DateFormat df = patchYear(DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.MEDIUM));
-		DATE_TIME_FORMATTER_HOLDER = new FormatThreadLocal<>(df);
+		DateFormat df = new SimpleDateFormat("MM/dd/yyyy h:mm:ss:SSS a");
```

Comment:
This change is forcing US formatting on everyone...

Reply:
@thegreystone  Can you please suggest a better way to make it generic for every user and still have the timestamp showing the ms part i.e. SSS?